### ISSUE TITLE
PumpkinBomb (Scarecrow)

### ISSUE THREAD

**Issue by ghost** [dniym/Bending#16]
**Describe the bug**
the move states that holding shift will shoot a rotten pumpkin bomb. 

**To Reproduce**
hold sneak on pumpkin bomb, nothing happens

**Expected behavior**
Should fire a missile or something if you hold sneak for long enough

**Screenshots**

**Additional Information**
Add any other information about the problem here.

**Comment by dniym** [comment]
Actually tapping sneak should drop a pumpkin on the ground that explodes if a player/(maybe mob) gets near it.  Once one is on the ground you can charge shift to launch please try this method and see if it still has an issue.

**Comment by ghost** [comment]
Just tested it out, perhaps the instructions weren't super clear. Also, the range seems super short on that one. Pumpkin missile would disappear after about 2 blocks.

**Comment by dniym** [comment]
Yeah I also tested this one and the pumpkin does launch but goes away almost instantly will work on fixing this and look over the instructions
